### PR TITLE
Updating Processing of External Datasets

### LINKED ISSUE #109: Determine which genes were removed from the GTEx Signature Transformation

### PR DESCRIPTION
**Pull request by gwaygenomics**
Closes #109

### CODE REVIEW

File: 8.gtex-interpret/scripts/nbconverted/2A.download-neutrophil-data.py [hunk 1]
```diff
@@ -53,6 +53,7 @@
 
 # Load Data
 geo_df = pd.read_excel(path, index_col=0, skiprows=1)
+geo_df = geo_df[geo_df.symbol != 'N\A']
```
Comment (ajlee21) [inline 8.gtex-interpret/scripts/nbconverted/2A.download-neutrophil-data.py]: is this the same as dropna?

File: 8.gtex-interpret/scripts/utils.py [hunk 2]
```diff
@@ -47,24 +48,24 @@ def load_weight_matrix(dataset, z_dim, seed, shuffled=False):
     return weight_df
 
 
-def load_enrichment_results(dataset, z_dim, metaedge, algorithm=None,
-                            feature=None, seed=None, shuffled=False):
+def load_enrichment_results(
+    dataset, z_dim, metaedge, algorithm=None, feature=None, seed=None, shuffled=False
```
Comment (ajlee21) [inline 8.gtex-interpret/scripts/utils.py]: minor,  you could create newlines to make this line shorter

File: 8.gtex-interpret/scripts/nbconverted/2A.download-neutrophil-data.py (hunk 1 above)
Comment (gwaygenomics) [reply 8.gtex-interpret/scripts/nbconverted/2A.download-neutrophil-data.py]: No, unfortunately this dataset had some strange symbols

File: 8.gtex-interpret/scripts/utils.py (hunk 2 above)
Comment (gwaygenomics) [reply 8.gtex-interpret/scripts/utils.py]: :+1:

**Comment by gwaygenomics** [comment]
Thanks for comments @ajlee21 - I made the minor fix in the most recent commit. I will merge!

**Pull request merged**